srcpkgs/quakespasm/template

```diff
@@ -13,14 +13,19 @@ license="GPL-2.0-or-later"
 homepage="http://quakespasm.sourceforge.net/"
 changelog="http://quakespasm.sourceforge.net/Quakespasm.html#s5"
 distfiles="${SOURCEFORGE_SITE}/quakespasm/Source/quakespasm-${version}.tar.gz"
-checksum=49a53e451d5e250982a855a8c9d2694c12397f01bd6fad0dea9e3f57f443b5cc
+checksum=b578d6ce4a4f7f4e26a24458f18c4bcc8d3854ae6252eb9982ee81d95e50180e
 
 case "$XBPS_TARGET_MACHINE" in
 	arm*) makedepends+=" MesaLib-devel"
 esac
 
+post_build() {
+	sed -e 's/Icon=net.sourceforge.quakespasm.Quakespasm/Icon=quakespasm/' \
```

Comment:
use vsed

Reply:
I have updated this PR to use vsed as requested.